### PR TITLE
Log task start time to verify delays and modify the loop to skip intermediate state

### PR DESCRIPTION
**Pull request by bhavanisn**
This add few debug information to collect in order to debug further the issue https://github.com/eclipse-openj9/openj9/issues/14469 which is not reproducible locally. The issue happens due to the intermediate state set in between the testcase from Task A to Task B and exits. This commit will also let the testcase run one another time so it will skip the intermediate state.

Signed-off-by: Bhavani SN (bhavani.sn@ibm.com)

### CODE REVIEW

**Comment by bhavanisn** [comment]
@dsouzai @llxia please review

**Comment by dsouzai** [comment]
For posterity:

> This commit will also let the testcase run one another time so it will skip the intermediate state.

The reason for doing another iteration is because if `goo()` returns a malformed string a second time, then it truly is a bug and will trigger the assert, since the state will remain `?`. On the other hand, forcing another iteration will ensure if that if the state was still `?` at the time `keepOnGoing` was set to `false`, we ensure that the state gets set to `B` before terminating.

In other words, even though it helps with debugging the linked issue, it is still something that the test should do in case the main cause of the issue is timing related.

File: test/functional/JIT_Test/src/jit/test/tr/chtable/VirtualGuardTest.java [hunk 1]
```diff
@@ -91,7 +95,7 @@ private char bar() {
 	}
 	
 	private int goo() {
-		return (f.method() + f.method() + f.method() + f.method());
+                return (f.method() + f.method() + f.method() + f.method());
```
Comment (dsouzai) [inline test/functional/JIT_Test/src/jit/test/tr/chtable/VirtualGuardTest.java:98]: For some reason the tabs got replaced with spaces here making it inconsistent with the rest of the file.

**Review by dsouzai — CHANGES_REQUESTED** [review]
Looks fine to me; minor change requested.

File: test/functional/JIT_Test/src/jit/test/tr/chtable/VirtualGuardTest.java (hunk 1 above)
Comment (bhavanisn) [reply test/functional/JIT_Test/src/jit/test/tr/chtable/VirtualGuardTest.java]: Fixed it !

**Comment by dsouzai** [comment]
I'll let @llxia review and merge this since it's a testing change.

**Comment by llxia** [comment]
https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/3063/

**Review by llxia — APPROVED** [review]
LGTM
